src/unicodeMake.js

```diff
@@ -11,7 +11,7 @@ const target = path.join(__dirname, 'unicodeSymbols.js');
 const targetMtime = fs.statSync(target).mtime;
 if (fs.statSync(__filename).mtime <= targetMtime && fs.statSync(
         path.join(__dirname, 'unicodeAccents.js')).mtime <= targetMtime) {
-    return;
```

Comment:
Wallaby compiles this file and complained about this `return`.  I'm kind of surprised eslint didn't pick this up sooner.  The issue is that it's a `return` that appears outside of a function.

Reply:
Both versions work in Node because the top-level script is wrapped in a function. But it is an invalid "ECMAScript program". [Weird](https://stackoverflow.com/questions/28955047/why-does-a-module-level-return-statement-work-in-node-js). Anyway, fine to change.